app/views/account/weeklyRenew.scala.html

```diff
@@ -57,7 +58,7 @@ <h3 class="mma-section__header">Renew your subscription</h3>
                 </p>
 
                 <div class="js-weekly-renew"
-                data-email="@contact.email"
+                data-email="@if(subscription.readerType == ReaderType.Direct) {@contact.email} else {}"
```

Comment:
Need to remember to remove this once you get the email from Zuora's Bill To instead of the Salesforce contact.